apps/fluent-tester/package.json

```diff
@@ -28,10 +28,12 @@
   "dependencies": {
     "@fluentui-react-native/android-theme": ">=0.3.1 <1.0.0",
     "@fluentui-react-native/apple-theme": ">=0.6.1 <1.0.0",
+    "@fluentui-react-native/button": ">=0.17.2 <1.0.0",
```

Comment:
This should be included with line 45 here, "@fluentui/react-native"

Reply:
Updated in `30c439e`

Reply:
Just kidding

ERR! [11:48:57 AM] x Missing dependency on @fluentui-react-native/button
ERR! [11:48:57 AM] x -- /Users/chripear/git/fluentui-react-native/apps/fluent-tester/src/FluentTester/TestComponents/NativeDatePicker/NativeDatePickerTest.tsx

Reply:
Huh, well that's weird. I guess it doesn't like transitive dependencies. I guess it's also better to be explicit 🤷🏾
Sorry about that...